src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -247,6 +418,16 @@ partial class BoundFixedStatement : IFixedStatement
         IStatement IFixedStatement.Body => this.Body;
 
         protected override OperationKind StatementKind => OperationKind.FixedStatement;
+
+        public override void Accept(IOperationVisitor visitor)
+        {
+            visitor.VisitFixedStatement(this);
+        }
+
+        public override TResult Accept<TResult>(IOperationVisitor<TResult> visitor)
+        {
+            return visitor.VisitFixedStatement(this);
+        }
     }
 
     partial class BoundUsingStatement : IUsingWithDeclarationStatement, IUsingWithExpressionStatement
```

Comment:
Consider wrapping `BoundUsingStatement` with an object that implements exactly one interface. Or merge the `IUsingWith...` interfaces into one, with optional `Value` and `Declarations` properties.